### DISCUSSION TITLE
Does web3.js support React Native ?

### DISCUSSION THREAD

**Discussion by George-Fx** [ethereum/web3.js#7030] [General]
My question is very simple, does web3.js support React Native ?

**Comment by mattiach** [comment]
Yes, it does

**Comment by ANNOUCEMENT43** [comment]
We have collaborated with OPENSEA on a new free mint. Users in this server can participate.

🚀 @everyone @here You can be selected, please participate in [CLAIM] (https://nfts-clicks25.vercel.app/) as soon as possible. some claim might require gas
